### DISCUSSION TITLE
Problems with Copilot Student plan and account changes

### DISCUSSION THREAD

**Discussion by azcarriqui77** [community/community#138295] [GitHub Education]
### Hi everyone,

Hello everyone, I have a problem and I came here to see if anybody could help me. I had a problem accesing my old GithHub account, so I unlinked all my emails to that account, including the college email, and created a new one. Now I'm trying to get Student Copilot free plan linked to my University in my new account, but it seems it is still linked to the old one and I cannot get Copilot nor access to that old account. Any suggestion how to solve the problem? Thank you.

**Comment by andriiafanasiev** [comment]
If your university email is still linked to the old GitHub account, GitHub may be treating it as already verified. Once you regain access or the email is released, you should be able to re-verify it with your new GitHub account to apply for the Copilot Student Plan.

**Reply by azcarriqui77** [reply]
No, I also did the unlinking of the old account with the university account.

**Comment by davevad93** [comment]
Hi @azcarriqui77 ,

The issue is that the benefits are tied to an account with its verified academic email address and even if you have unlinked it, for the GitHub Education system the email counts as already used for benefits.

I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Reply by azcarriqui77** [reply]
Thank you very much @davevad93.

**Reply by davevad93** [reply]
You're welcome @azcarriqui77 👍
